go.mod

```diff
@@ -2,21 +2,33 @@ module github.com/rancher/lasso
 
 go 1.20
 
+replace (
+	k8s.io/api => k8s.io/api v0.28.6
+	k8s.io/apimachinery => k8s.io/apimachinery v0.28.6
+	k8s.io/client-go => k8s.io/client-go v0.28.6
+)
+
 require (
 	github.com/golang/mock v1.6.0
+	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_golang v1.16.0
+	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
 	k8s.io/api v0.28.6
 	k8s.io/apimachinery v0.28.6
-	k8s.io/client-go v0.28.6
+	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/klog v1.0.0
+	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2
+	modernc.org/sqlite v1.27.0
```

Comment:
Is it possible to use the current latest version instead [v1.29.10](https://gitlab.com/cznic/sqlite/-/tags/v1.29.10)?